can/io/asc.py

```diff
@@ -43,120 +43,156 @@ def __init__(self, file, base="hex"):
         """
         super().__init__(file, mode="r")
         self.base = base
+        self._converted_base = self._check_base(base)
+        self.date = None
+        self.timestamps_format = None
+        self.internal_events_logged = None
```

Comment:
This doesn't seem to be used anywhere. It seems to just indicate the presence of the following line:

```
internal events logged
```

Can you help me understand why this needs to be exposed?

Reply:
@karlding The line can be either "internal events logged" or "no internal events logged". It may be useful in the future as we add support for more kinds of frames/events in the AscReader.